coresimd/mod.rs

```diff
@@ -86,7 +86,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "aarch64", dox))]
     #[doc(cfg(target_arch = "aarch64"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```

Comment:
The feature name that we could re-use is `aarch64_target_feature`, the alternative would be to add a new feature to rustc here (`aarch64_arch`?)